proposal_guide.md

```diff
@@ -0,0 +1,164 @@
+  assessment of implementation quality and interoperability.
+
+* Tests must be fully automated and included in Chrome, Edge, Firefox
+  and Safari desktop runs on wpt.fyi.
+
+Where there is insufficient test coverage during the proposal period,
+proposers can provide an indication of how they expect to achieve the
+necessary test coverage before the final focus areas are
+picked. Individual participants will judge how credible they consider
+these plans to be, and may choose to focus on proposals where tests
+are already available.
+
+### Ongoing Standards Work
+
+Interop is not a venue for performing standards work. Proposals which
+are not backed by an existing standard in a recognized standards venue
+are highly unlikely to be accepted.
+
+Proposals may cover areas which still have some unresolved standards
+questions. In these cases it's likely that one of the following will
+be required:
+
+* The areas of ongoing discussion are explicitly excluded from the
+  focus area.
+
+* The areas of ongoing discussion are understood to be highly
+  significant in terms of interoperability, have tentative tests, and
+  participants agree that reaching resolution on the outstanding
+  questions is possible either before the Interop period starts or
+  early in the Interop period.
+
+## Additional Signals
+
+### Standards Positions
+
+Interop is not intended as a venue to reach agreement on whether a
+feature is suitable for the web platform.
+
+Proposals covering features where participants have already indicated
+significant concerns, for example through standards-position
+repositories or similar public venues, are highly unlikely to be
+accepted. Proposers may of course work with participants to address
+their existing concerns before making an Interop proposal.
+
+
+### Site Breakage and Workaround
+
+This includes any evidence of interoperability problems affecting web
+developers today, for example:
+
+* Sites that are broken in some browsers due to interoperability
+  problems.
+
+* Evidence of sites or libraries needing to work around the browser
+  differences in order to function correctly across multiple browsers.
+
+### Size and Current State of the Feature
+
+Whether or not the feature is already widely implemented, and the
+confidence we have on full implementation being achievable:
+
+* Features that are already widely implemented but have behavior
+  differences may be considered more pressing than implementing new
+  functionality.
+
+* Features that are already present in multiple browser engines may be
+  a higher priority than recently standardized features that are not
+  yet implemented.
+
+* Use counter data might be considered, if it's showing a clear trend
+  and can be cleanly attributed to a real increase in use of the
+  feature. Past experience shows that absolute numbers in use counter
+  data can be misleading, so this is unlikely to carry much weight on
+  its own.
+
+### Bug Trackers
```

Comment:
I think "Browser Bugs" will read better in a ToC inlined into the proposal templates:

```suggestion
### Browser Bugs
```